drivers/storage/vbox/vbox_driver.go

```diff
@@ -0,0 +1,658 @@
+package vbox
+
+import (
+	"io/ioutil"
+	"net"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/akutz/gofig"
+	"github.com/akutz/goof"
+	vbox "github.com/appropriate/go-virtualboxclient/virtualboxclient"
+	"github.com/emccode/libstorage/api/registry"
+	"github.com/emccode/libstorage/api/types"
+	"github.com/emccode/libstorage/api/types/context"
+	"github.com/emccode/libstorage/api/types/drivers"
+	"github.com/emccode/libstorage/drivers/storage/vbox/executor"
+	"github.com/emccode/rexray/core/errors"
```

Comment:
Hi @clintonskitson,

@vladimirvivien may have not even done that on purpose. Atom's Go integration will auto-resolve package imports and use the most ideal path it can find upon saving a file. Just find out why it's importing that and dupe that functionality here.